Announce the launch of Skolar for the scikit-learn community (#217)
scikit-learn/blog: _posts/2025-30-06-probabl-skolar.md

---
#### Blog Post Template ####

#### Post Information ####
title: "Skolar: an open-source initiative to democratize open data science"
date: June 30, 2025

#### Post Category and Tags ####
# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
categories:
  - Updates
tags:
  - Open Source
  - Machine Learning

#### Featured Image ####
featured-image: BSD_watermark.svg

#### Author Info ####
# Can accomodate multiple authors
# Add SQUARE Author Image to /assets/images/author_images/ folder
postauthors:
  - name: Pénélope Gittos
    website: https://www.linkedin.com/in/gittospenelope-data-analyst-growth-bilingual/
---
<div>
  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
  {% include postauthor.html %}
</div>
https://skolar.probabl.ai/

> The scikit-learn project always puts efforts on education to build and nurture a strong vibrant open-source community. The goal is straightforward: give everyone, everywhere, the tools they need to easily grasp, engage with, and meaningfully contribute to data science using open-source software. This mission is shared and actively supported by Probabl, a company that helps maintain scikit-learn by employing many of its core contributors and investing in its long-term sustainability. With their support and a deep commitment from the community, we continue building bridges between research, software, and education.

Reviewer: Maybe we can link to Probabl since it's first appearance:

```suggestion
The scikit-learn project always puts efforts on education to build and nurture a
strong vibrant open-source community. The goal is straightforward: give
everyone, everywhere, the tools they need to easily grasp, engage with, and
meaningfully contribute to data science using open-source software. This mission
is shared and actively supported by [Probabl](https://probabl.ai/), a company
that helps maintain scikit-learn by employing many of its core contributors and
investing in its long-term sustainability. With their support and a deep
commitment from the community, we continue building bridges between research,
software, and education.
```

> When the [Inria scikit-learn MOOC](https://inria.github.io/scikit-learn-mooc/)(Massive Open Online Course) first went live, our community got a front-row seat to the amazing impact of practical, accessible and open learning. More than 40,000 people worldwide have jumped into these courses, clearly highlighting the demand for organized, hands-on resources that blend theory with real-world practice.
> 
> Today, [Probabl](https://probabl.ai/) is excited to introduce Skolar, a new, fully open-source educational initiative, built directly from your feedback and all the lessons we've learned along the way. Developed by the maintainers and core developers of scikit-learn, Skolar is designed specifically for data science practitioners, offering hands-on, high-quality learning resources grounded in real-world applications and open-source values.

Reviewer: I rephrased a little bit to better make clear that those behind the Inria Mooc are the same persons now behind Skolar.

```suggestion
When the [Inria scikit-learn MOOC](https://inria.github.io/scikit-learn-mooc/)
(Massive Open Online Course) first went live, our community got a front-row seat
to the amazing impact of practical, accessible and open learning. Created by
several core developers and maintainers of scikit-learn—now working at
Probabl—the MOOC has reached over 40,000 learners worldwide, clearly
highlighting the demand for organized, hands-on resources that blend theory with
real-world practice.

Today, Probabl is excited to introduce Skolar, a new, fully open-source
educational initiative, built directly from your feedback and all the lessons
we've learned along the way. Developed and extended by those same core
developers of scikit-learn, Skolar is designed specifically for data science
practitioners, offering hands-on, high-quality learning resources grounded in
real-world applications and open-source values.
```

> Skolar exists to boost our shared values: openness, teamwork, and practicality. It offers clear, interactive tutorials and structured courses carefully designed to match industry challenges and specialized use-cases. But even more importantly, it captures the true spirit of open source: encouraging collaboration, peer-to-peer learning, and guidance from experts.
> 
> Right now, we’re just at the beginning. Today, you can dive into our Scikit-learn Associate Practitioner online course, adapted from the popular Inria MOOC but enhanced with new material on unsupervised learning, especially clustering.
> 
> The next stages, professional and expert levels, will launch soon. We’ll also add more courses covering other open-source libraries such as skrub (for data wrangling), hazardous (for survival analysis), and fairlearn (for fairness). Additionally, our scikit-learn team is planning to create  industry-specific modules tackling real-world needs in fields like healthcare, finance, medicine, and beyond.
> 
> At its core, Skolar is about empowering people through education, driven entirely by our passion for openness and collaboration. We firmly believe that true open data science begins with community-built learning resources.
> We warmly welcome you, whether you're a contributor, learner, teacher, or just someone curious, to join us. Help shape Skolar’s future and support open-source education in data science.

Reviewer: No changes here, only for formatting:

```suggestion
Skolar exists to boost our shared values: openness, teamwork, and practicality.
It offers clear, interactive tutorials and structured courses carefully designed
to match industry challenges and specialized use-cases. But even more
importantly, it captures the true spirit of open source: encouraging
collaboration, peer-to-peer learning, and guidance from experts.

Right now, we’re just at the beginning. Today, you can dive into our
Scikit-learn Associate Practitioner online course, adapted from the popular
Inria MOOC but enhanced with new material on unsupervised learning, especially
clustering.

The next stages, professional and expert levels, will launch soon. We’ll also
add more courses covering other open-source libraries such as skrub (for data
wrangling), hazardous (for survival analysis), and fairlearn (for fairness).
Additionally, our scikit-learn team is planning to create  industry-specific
modules tackling real-world needs in fields like healthcare, finance, medicine,
and beyond.

At its core, Skolar is about empowering people through education, driven
entirely by our passion for openness and collaboration. We firmly believe that
true open data science begins with community-built learning resources. We warmly
welcome you, whether you're a contributor, learner, teacher, or just someone
curious, to join us. Help shape Skolar’s future and support open-source
education in data science.
```
